src/extractTriggers.js

```diff
@@ -5,28 +5,26 @@
 
 var extractTriggers = function(mod, triggers, prefix) {
   prefix = prefix || "";
-  for (var funcName in mod) {
-    if (mod.hasOwnProperty(funcName)) {
-      var child = mod[funcName];
-      if (typeof child === "function" && child.__trigger && typeof child.__trigger === "object") {
-        if (funcName.indexOf("-") >= 0) {
-          throw new Error(
-            'Function name "' + funcName + '" is invalid. Function names cannot contain dashes.'
-          );
-        }
+  for (var funcName of Object.keys(mod)) {
+    var child = mod[funcName];
+    if (typeof child === "function" && child.__trigger && typeof child.__trigger === "object") {
+      if (funcName.indexOf("-") >= 0) {
+        throw new Error(
+          'Function name "' + funcName + '" is invalid. Function names cannot contain dashes.'
+        );
+      }
 
-        var trigger = {};
-        for (var key in child.__trigger) {
-          if (child.__trigger.hasOwnProperty(key)) {
-            trigger[key] = child.__trigger[key];
-          }
+      var trigger = {};
+      for (var key in child.__trigger) {
+        if (child.__trigger.hasOwnProperty(key)) {
```

Comment:
Is this use of `hasOwnProperty` not susceptible to the same issue?

Reply:
It is not susceptible because it's built within `firebase-functions` but I will change this loop too just because `for of` looks better ;)